rlclientlib/vw_model/safe_vw.h

```diff
@@ -40,11 +40,15 @@ class safe_vw
 
   bool is_compatible(const std::string& args) const;
   bool is_CB_to_CCB_model_upgrade(const std::string& args) const;
+  const char* get_audit_data() const;
```

Comment:
Can you add a comment clarifying that a call to `init` or any of the other rank etc functions will invalidate this pointer